math-libs/CMakeLists.txt

```diff
@@ -193,3 +193,78 @@ therock_provide_artifact(rand
     run
   SUBPROJECT_DEPS rocRAND
 )
+
+#################################################################################
+# rocPRIM
+#################################################################################
+
+therock_cmake_subproject_declare(rocPRIM
+  EXTERNAL_SOURCE_DIR "rocPRIM"
+  BACKGROUND_BUILD
+  CMAKE_ARGS
+    -DHIP_PLATFORM=amd
+    -DROCM_PATH=
+    -DROCM_DIR=
+  COMPILER_TOOLCHAIN
+    amd-hip
+  RUNTIME_DEPS
+    hip-clr
+)
+therock_cmake_subproject_glob_c_sources(rocPRIM
+  SUBDIRS
+    .
+)
+therock_cmake_subproject_provide_package(rocPRIM rocprim lib/cmake/rocprim)
+therock_cmake_subproject_activate(rocPRIM)
+
+# therock_provide_artifact(prim
```

Comment:
Fine if you want to iterate on this more, but can you leave a TODO here and below?

Reply:
I (hopefully) added the missing peaces and we can therefore skip adding a TODO.

PTAL.